File: 7-day/volume-cylinder.py
from math import pi

def area_circle(r):
    area = pi * (r ** 2)
    return area


def volume_cylinder(a, h):
    volume = a * h
    return volume


if __name__ == '__main__':

    radio = float(input('Enter the radio: '))
    height = float(input('Enter the height of cylinder: '))
    area = area_circle(radio)
    volume = volume_cylinder(area, height)
    volume = round(volume, 1)

    print(f'Volume of cylinder = {volume} cubic units')

File: 8-day/volume-cylinder.js
const height = prompt('Enter height of cylinder: ')
const radio = prompt('Enter radio of cylinder: ')

function volumeCylinder(height, radio) {
  result = (Math.PI) * (radio ** 2) * (height)
  alert(`Result; ${result.toFixed(4)} m3`)
}

volumeCylinder(height, radio)